src/components/common/FirmContactInfo.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <ul class="contact-info pl-0">
+    <li class="contact-container">
+      <v-icon
+        class="contact-icon"
+        size="18px"
+      >
+        mdi-phone
+      </v-icon>
+      <span class="px-2">Canada and U.S. Toll Free:</span>
+      <a
+        href="tel:+1-877-370-1033"
+        class="contact-value"
+      >1-877-370-1033</a>
+    </li>
+    <li class="contact-container">
+      <v-icon
+        class="contact-icon"
+        size="18px"
+      >
+        mdi-phone
+      </v-icon>
+      <span class="px-2">Victoria Office:</span>
+      <a
+        href="tel:+1-250-370-1033"
+        class="contact-value"
+      >250-370-1033</a>
+    </li>
+  </ul>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component({})
+export default class RegistriesContactInfo extends Vue {}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.contact-container {
+  font-size: $px-15;
+  color: $gray7;
+  display: flex;
+
+  .contact-icon {
```

Comment:
Please Shaoyun if you can fix the indentation here and the class below.

Reply:
Please also stop importing the obsolete component, and the store getter and imports.

Reply:
> Please Shaoyun if you can fix the indentation here and the class below.

![image](https://github.com/bcgov/business-create-ui/assets/144159934/f12a69b5-699b-4ee1-85e7-7dd851975295)
I think the indentation should be that, `contact-icon` and `contact-value` class are under `class-container`

Reply:
> Please also stop importing the obsolete component, and the store getter and imports.

Do you mean removing this part?
![image](https://github.com/bcgov/business-create-ui/assets/144159934/ed638a9d-ad55-442c-b9e5-d4a1eb0f2a61)

Reply:
That's fine then. I was thinking you could make the classes separate (not nested) but in your case it also works. Resolving, thanks.